db/init.sql

```diff
@@ -94,52 +104,118 @@ prevent confusion about were they left out intentionally or by accident
-    (default, 'Composition2', 'composer2', 'composition2_eng', 'composition2_fin'),
-    (default, 'Composition3', 'composer3', 'composition3_eng', 'composition3_fin');
 
---concerts dummy data
+--The kids' concert performances. The same event three times with the only difference being the language. Also no performers mentioned.
+    (1, 'In Finnish', 'Suomeksi', NULL, NULL),
+    (2, 'In Finnish', 'Suomeksi', NULL, NULL),
+    (3, 'In English', 'Englanniksi', NULL, NULL),
 
-INSERT INTO concert (concertid, concertname, concertdate, consertstarttime, concertinfo_eng, concertinfo_fin, venueid)
-VALUES
-    (default, 'Concert1', '2022-03-20', '19:00', 'concert1info_eng', 'concert1info_fin', 1),
-    (default, 'Concert2', '2022-03-24', '18:30', 'concert2info_eng', 'concert2info_fin', 1),
-    (default, 'Concert3', '2022-03-22', '19:15', 'concert3info_eng', 'concert3info_fin', 1),
-    (default, 'Concert4', '2022-05-10', '14:00', 'concert4info_eng', 'concert4info_fin', 1);
+--TFF Youth performances
+
+    (4, 'Inside the French School - Warm Up', 'Inside the French School - lämmittely', 'sat1eng.txt', 'sat1fin.txt'),
+    (5, 'Masterclass: Sébastian Jacot', 'Mestarikurssi: Sébastian Jacot', 'sat2eng.txt', 'sat2fin.txt'),
+    (6, 'Masterclass: Jenny Villanen', 'Mestarikurssi: Jenny Villanen', 'sat3eng.txt', 'sat3fin.txt'),
+    (7, 'Recital: Piccolo Variations - Jenny Villanen', 'Resitaali: Piccolo Variations - Jenny Vilanen', 'sat4eng.txt', 'sat4fin.txt'),
+    (8, 'Masterclass: Jenny Villanen', 'Mestarikurssi: Jenny Villanen', 'sat5eng.txt', 'sat5fin.txt'),
+    (9, 'Masterclass: Alexis Roman', 'Mestarikurssi: Alexis Roman', 'sat6eng.txt', 'sat6fin.txt'),
+    (10, 'Recital: Young Talent -Fabian Egger', 'Resitaali: Young Talent -Fabian Egger', 'sat7eng.txt', 'sat7fin.txt'),
+    (11, 'Recital: An American in Paris - Beatriz Macías and Alexis Roman', 'Resitaali: An American in Paris - Beatriz Macías and Alexis Roman', 'sat8eng.txt', 'sat8fin.txt'),
+
+--Gala concert performances
 
--- programme dummy data
+    (12, '2022 Young Artist Competition Finals', 'Young Artist 2022 -kilpailun finaali', 'sun1eng.txt', 'sun1fin.txt'),
+    (13, 'Recital: New Generations - Astrid Bjelland', 'Resitaali: New Generations - Astrid Bjelland', 'sun2eng.txt', 'sun2fin.txt'),
+    (14, 'Masterclass: Sébastian Jacot', 'Mestarikurssi: Sébastian Jacot', 'sun3eng.txt', 'sun3fin.txt'),
+    (15, 'Recital: La flute enchantée - Sébastian Jacot', 'Resitaali: La flute enchantée - Sébastian Jacot', 'sun4eng.txt', 'sun4fin.txt'),
+    (16, 'Panel Discussion: In the Spotlight!', 'Paneelikeskustelu: Valokeilassa!', 'sun5eng.txt', 'sun5fin.txt'),
+    (17, 'JÄÄ//ICE Gala Concert', 'JÄÄ//ICE - Gaalakonsertti', 'sun6eng.txt', 'sun6fin.txt');
 
-INSERT INTO programme (concertid, artistid, compositionid, performancetime)
+--concerts data
+
+INSERT INTO concert (concertid, concertname, concertdate, consertstarttime, concertinfo_eng, concertinfo_fin, venueid)
```

Comment:
column concertendtime missing. Also, the name of the column consertstarttime is correct but there is a typo (consert insteed of concert like all the other names)

Reply:
Nice looking review with the relevant part of code in it :D

Reply:
But how did that not fail in the automatic testing? It makes the whole database structure and runs the init.sql.

Reply:
good question XD we don't call concert in a query so maybe we don't have any errors for now ?